node/nodeRunner.go

```diff
@@ -813,6 +812,8 @@ func (nr *nodeRunner) CreateManagedProcessComponents(
 		StartEpochNum:             managedBootstrapComponents.EpochBootstrapParams().Epoch(),
 		SizeCheckDelta:            configs.GeneralConfig.Marshalizer.SizeCheckDelta,
 		StateCheckpointModulus:    configs.GeneralConfig.StateTriesConfig.CheckpointRoundsModulus,
+		UserStatePruningQueueSize: configs.GeneralConfig.StateTriesConfig.UserStatePruningQueueSize,
+		PeerStatePruningQueueSize: configs.GeneralConfig.StateTriesConfig.PeerStatePruningQueueSize,
```

Comment:
I will ask for some refactor since you are changing in this area :)
Could you remove from ProcessComponentsFactoryArgs: 
- SizeCheckDelta
- StateCheckpointModulus
- UserStatePruningQueueSize
- PeerStatePruningQueueSize
- MaxComputableRounds
- NumConcurrentResolverJobs
- MinSizeInBytes
- MaxSizeInBytes
- EpochStart
these are all accessible from ProcessComponentsFactoryArgs.Config

Also you could remove also:
- ValidatorPubkeyConverter
- HeaderIntegrityVerifier
- EconomicsData
- StartEpochNum
- ShardCoordinator
- RoundHandler
- EpochStartNotifier
- Rater
- RatingsData
- Indexer
- TpsBenchmark
and add instead a BootstrapComponentsHolder that could be initialized here with managedBootstrapComponents and a StatusComponentsHolder initialized with managedStatusComponents, then all the above will be accessible from the available components holders

Reply:
Done.